qdrant_client/qdrant_client.py

```diff
@@ -21,6 +21,14 @@ class QdrantClient:
     It combines interface classes and endpoint implementation.
     Additionally, it provides custom implementations for frequently used methods like initial collection upload.
 
+    .. note::
```

Comment:
Maybe another note here for union types of arguments, automatic conversion and some performance tips?